l1-contracts/deploy-scripts/upgrade/EcosystemUpgrade.s.sol

```diff
@@ -112,25 +111,20 @@ contract EcosystemUpgrade is Script, DeployL1Script {
         // for facilitating partially trusted, but not critical tasks.
         address ecosystemAdminAddress;
         uint256 governanceUpgradeTimerInitialDelay;
-        uint256 newProtocolVersion;
+        //uint256 newProtocolVersion;
         uint256 oldProtocolVersion;
         address oldValidatorTimelock;
     }
 
     // solhint-disable-next-line gas-struct-packing
     struct NewlyGeneratedData {
+        // TODO: why this is here - and not using "regular" generateddata.
```

Comment:
the gateway upgrade reused the L2GenesisUpgradeHelper/L2GatewayUpgradeHelper, so it needed this fixedForceDeployments field. in general it should not be needed